.github/workflows/test.yml

```diff
@@ -30,15 +30,13 @@ jobs:
         with:
           go-version: ${{ matrix.go }}
         id: go
-      - name: Set up GOPATH
+      - name: Set up INSTALL_DIRECTORY
         run: |
-          go version
-          echo "GOPATH=${{ github.workspace }}" >> $GITHUB_ENV
-          echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
+
```

Comment:
I left this space because my VSCode IDE only does correct syntax highlighting for the comment if I have this space here... let me know if this is undesirable!